packages/renderer/src/lib/onboarding/OnboardingEmbeddedComponent.svelte

```diff
@@ -0,0 +1,50 @@
+<script lang="ts">
```

Comment:
I'm not sure why it's called an embedded component

it's a component, then you might include existing svelte components but it's implementation specific.

so it looks like it's more OnboardingComponent and then you have a list of supported components that may or not include existing parts